test/config/test_configtypes.py

```diff
@@ -1231,34 +1217,32 @@ def test_validate_empty_none_ok(self):
         t = configtypes.FontFamily(none_ok=True)
         t.validate('')
 
-    def test_validate_valid(self):
+    @pytest.mark.parametrize('val', TESTS)
+    def test_validate_valid(self, val):
         """Test validate with valid values."""
-        for val in self.TESTS:
-            with self.subTest(val=val):
-                self.t.validate(val)
+        self.t.validate(val)
 
-    def test_validate_invalid(self):
+    @pytest.mark.parametrize('val', INVALID)
+    def test_validate_invalid(self, val):
         """Test validate with invalid values."""
-        for val in self.INVALID:
-            with self.subTest(val=val):
-                with self.assertRaises(configexc.ValidationError, msg=val):
-                    self.t.validate(val)
+        with pytest.raises(configexc.ValidationError, msg=val):
```

Comment:
I think the `msg=val` here does nothing (and is unneeded, as pytest prints the parameter anyways I think)